requirements-dev.txt

```diff
@@ -23,3 +23,4 @@ sphinx_rtd_theme
 sphinxcontrib-mermaid
 twine
 wheel
+sympy
```

Comment:
I don't think sympy needs to go here, too. It only needs to be in `requirements.txt`
```suggestion
```